action.yml

```diff
@@ -39,6 +39,20 @@ runs:
       uses: OctopusDeploy/install-octopus-cli-action@v3
       with:
         version: '*'
+    - name: Set environment variables
```

Comment:
Remove reading from octopus.yaml from everywhere. We'll settle with just reading from env variables